doc.md

```diff
@@ -811,7 +811,7 @@ Apply filter pattern after each keystroke during filtering.
 ## info ([]string)  (default ``)
 
 A list of information that is shown for directory items at the right side of the pane.
-Currently supported information types are `size`, `time`, `atime`, and `ctime`.
+Currently supported information types are `size`, `time`, `atime`, `ctime`, `mode`, `user` and `group`.
```

Comment:
Just wondering, do you think instead of using `mode`, something like `perm` or `permissions` is more intuitive as the name here?

Reply:
Yeah I agree, I will change it to perm.